.changeset/short-cycles-film.md

```diff
@@ -0,0 +1,8 @@
+---
+'@sphinx-labs/contracts': minor
+'@sphinx-labs/plugins': minor
+'@sphinx-labs/core': minor
```

Comment:
The issue with syncing in the deploy CLI command is that the user then cannot really offboard from our platform.

To be honest, this sort of question is insignificant enough that I really wouldn't worry about it. I would like to get this change out sooner rather than later, if the current way the deploy cli command works is an issue then I'm happy to change it. But I don't think it's worth spending much time on this feature.